README.md

```diff
@@ -1,2 +1,8 @@
 # priority-care-modeling
 Predicting Wait Times for Priority Medical Procedures in Canada: Analyzing the Role of Health Workforce Distribution Across Provinces
+
+
+
+### Note from 26/7/2025
+- Aggregating data for each province based on the year given
+- Let's skip territories because it does not exist on the Table 1  Wait times for priority procedures, by province and Canada, 2008 to 2024 data
```

Comment:
Oh we missed one point where we said
"ewygvcedtywfvedghjie3qtdcftwgvytf2dy ecdiyugedhewuodycevtwygqh"